File: Data Structure with C/fibonacciSeries.c
#include <stdio.h>
void fibonacci(int n) {
    int first = 0, second = 1, next, i;
    printf("Fibonacci Series up to %d terms: \n", n);
    printf("%d, %d, ", first, second);
    for (i = 2; i < n; ++i) {
        next = first + second;
        printf("%d", next);
        if (i < n - 1) {
            printf(", ");
        }
        first = second;
        second = next;
    }
    printf("\n");
}
int main() {
    int terms;
    printf("Enter the number of terms in Fibonacci series: ");
    scanf("%d", &terms);
    fibonacci(terms);
    return 0;
}

File: Java/FibonacciSeries.java
import java.util.Scanner;
public class FibonacciSeries{
    public static void main(String[] args) {
        int num;
        int n1=0,n2=1,n3;
        Scanner fb=new Scanner(System.in);
        System.out.print("Enter the number: ");
        num=fb.nextInt();
        System.out.println("The fibonacci Series:-");
        System.out.print(n1+"  "+n2);
        for(int i=2;i<num;i++){ //loop start from 2 bcz 0 and 1 already printed
            n3=n1+n2;
            System.err.print("  " + n3);
            n1=n2;
            n2=n3;
        }
        fb.close();
    }
}
